src/mission_ui.cpp

```diff
@@ -0,0 +1,160 @@
+#include "game.h"
+#include "player.h"
+#include "output.h"
+#include "input.h"
+#include "mission.h"
+#include "weather.h"
+#include "calendar.h"
+
+#include <vector>
+#include <string>
+
+
+void game::list_missions()
+{
+    WINDOW *w_missions = newwin( FULL_SCREEN_HEIGHT, FULL_SCREEN_WIDTH,
+                                 ( TERMY > FULL_SCREEN_HEIGHT ) ? ( TERMY - FULL_SCREEN_HEIGHT ) / 2 : 0,
+                                 ( TERMX > FULL_SCREEN_WIDTH ) ? ( TERMX - FULL_SCREEN_WIDTH ) / 2 : 0 );
+
+    int tab = 0;
```

Comment:
Not mandatory for this PR but nice to have thing:
plain numbers are not self-explanatory, so it will be much better to use ```enum``` here. Something like
```
enum mission_tab_mode {
    TAB_ACTIVE,
    TAB_COMPLETED,
    TAB_FAILED
};
```
So lines like ```if( tab != 1 ) {``` becomes to ```if( tab != TAB_COMPLETED ) {``` which is much easier to understand.

Example of this approach is available here:
https://github.com/CleverRaven/Cataclysm-DDA/blob/master/src/bionics_ui.cpp#L20